Refresh.GameServer/Endpoints/Game/DataTypes/Response/GameUserResponse.cs

```diff
@@ -103,5 +99,46 @@ private void FillInExtraData(GameUser old, TokenGame gameVersion)
             TokenGame.Website => "0",
             _ => throw new ArgumentOutOfRangeException(nameof(gameVersion), gameVersion, null),
         };
+
+        //Fill out the used slots
+        switch (gameVersion)
+        {
+            case TokenGame.LittleBigPlanet3: {
+                //Match all LBP3 levels
+                this.UsedSlotsLBP3 = old.PublishedLevels.Count(x => x._GameVersion == (int)TokenGame.LittleBigPlanet3);
+                this.FreeSlotsLBP3 = 100 - this.UsedSlotsLBP3;
+                //Fill out LBP2/LBP1 levels
+                goto case TokenGame.LittleBigPlanet2;
+            }
+            case TokenGame.LittleBigPlanet2: {
+                //Match all LBP2 levels
+                this.UsedSlotsLBP2 = old.PublishedLevels.Count(x => x._GameVersion == (int)TokenGame.LittleBigPlanet2);
+                this.FreeSlotsLBP2 = 100 - this.UsedSlotsLBP2;
+                //Fill out LBP1 levels
+                goto case TokenGame.LittleBigPlanet1;
+            }
+            case TokenGame.LittleBigPlanetVita: { 
+                //Match all LBP Vita levels
+                this.UsedSlotsLBP2 = old.PublishedLevels.Count(x => x._GameVersion == (int)TokenGame.LittleBigPlanetVita);
+                this.FreeSlotsLBP2 = 100 - this.UsedSlotsLBP2;
+                //Fill out LBP1 levels
+                goto case TokenGame.LittleBigPlanet1;
```

Comment:
Can probably skip this entirely, Vita is it's own little world

Reply:
Ah, for some reason i remember being told Vita could load LBP1 levels, in that case i'll just `break` out